src/tool/subcommands/index_cmd.rs

```diff
@@ -80,9 +77,24 @@ impl IndexCommands {
 
                 let state_manager = Arc::new(StateManager::new(chain_store.clone(), chain_config)?);
 
+                let head_ts = chain_store.heaviest_tipset();
+
+                println!("Database path: {}", db_root_dir.display());
+                println!("From epoch:    {}", from.unwrap_or_else(|| head_ts.epoch()));
+                println!("To epoch:      {}", to);
                 println!("Head epoch:    {}", head_ts.epoch());
 
-                backfill_db(&state_manager, &head_ts, *to).await?;
```

Comment:
It always backfills index from CAR db head instead of the epoch specified in `--from`